Tools/Scripts/libraries/webkitscmpy/webkitscmpy/program/pull_request.py

```diff
@@ -317,13 +317,21 @@ def create_pull_request(cls, repository, args, branch_point, callback=None, unbl
             sys.stderr.write('Checks have failed, aborting pull request.\n')
             return 1
 
-        issue = None
+        issues = []
         commits = list(repository.commits(begin=dict(hash=branch_point.hash), end=dict(branch=repository.branch)))
         for line in commits[0].message.split() if commits[0] and commits[0].message else []:
             issue = Tracker.from_string(line)
             if issue:
+                issues.append(issue)
+            if not line:
                 break
 
+        radars = [issue for issue in issues if isinstance(issue.tracker, radar.Tracker)]
+        not_radars = [issue for issue in issues if not isinstance(issue.tracker, radar.Tracker)]
+        if radars and not_radars and radars[0].tracker.radarclient():
+            not_radars[0].cc_radar(radar=radars[0])
+        issue = issues[0] if issues else None
```

Comment:
nit: Can use `next` instead of list comprehension since we are only working with the first value
```suggestion
        radar = next(issue for issue in issues if isinstance(issue.tracker, radar.Tracker), None)
        issue = next(issue for issue in issues if not isinstance(issue.tracker, radar.Tracker), None)
        if radar and issue and radar.tracker.radarclient():
            issue.cc_radar(radar=radar)
```

Reply:
> nit: Can use `next` instead of list comprehension since we are only working with the first value

Probably shouldn't use `radar` since that collides with an import, but yes, this is a good idea, will change.